### DISCUSSION TITLE
GitHub Copilot not in Visual Studio 2022 MarketPlace

### DISCUSSION THREAD

**Discussion by drxvb** [community/community#47827] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Marketplace don't have GitHub CoPilot,
Tried to login to github and also it is not there
Open VSCode Extensions and found it there, but not in Normal Visual studio 2022 or Visual Studio 2019 Marketplace,

where I can find VISX file to install manually,

Best regards,

**Comment by drxvb** [comment, marked as answer]
I found the solution, you shall update from 17.0.0 to 17.2+

**Comment by cehfisher** [comment]
Hi @drxvb - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/47827#discussioncomment-5014988), but it also looks like your issue was mistakenly added to the accessibility discussion board.

@queenofcorgis could you please move this to the correct category and close this issue? Thanks!

**Comment by dexoron** [comment]
(This is Russian Language)
Еще может быть проблемма с не официальной версией VSCode
GitHub Copilot есть только в официальной версии
на Linux - версия с Flatpak не подойдет
можно скачать через yay
yay -S visual-studio-code-bin
это для ArchLinux, на других - другая команда
